web/modules/custom/joinup_search/config/install/facets.facet.type.yml

```diff
@@ -22,6 +22,7 @@ widget:
     soft_limit_settings:
       show_less_label: 'Show less'
       show_more_label: 'Show more'
+    arbitrary_facet_plugin: _none
```

Comment:
What is the reason for this change. This made tests to fail.

Reply:
Looks like config export put this line automatically during the export. Don't think that it relates to my current feature so will remove it.